nuxeo-mongodb-directory/src/main/resources/OSGI-INF/MongoDBDirectoryFactory.xml

```diff
@@ -1,51 +1,55 @@
 <?xml version="1.0"?>
 <component name="org.nuxeo.directory.mongodb.MongoDBDirectoryFactory">
 
-    <implementation class="org.nuxeo.directory.mongodb.MongoDBDirectoryFactory" />
+  <implementation class="org.nuxeo.directory.mongodb.MongoDBDirectoryFactory"/>
 
-    <require>org.nuxeo.ecm.directory.DirectoryServiceImpl</require>
+  <require>org.nuxeo.ecm.directory.DirectoryServiceImpl</require>
 
+  <documentation>
+    MongoDB-based implementation for NXDirectory
```

Comment:
-> `MongoDB-based implementation of a Directory.`

[NXDirectory is a very old name we used in the past but is not very descriptive anymore]